docs/assets/sample_ssp.md.jinja

```diff
@@ -0,0 +1,37 @@
+# SSP template 
+<!--
+This produces an SSP analog. 
+-->
+
+# Frontmatter
+<!-- This includes frontmatter.md showing how arbitrary prose can be inserted into a document --> 
+{% md_clean_include 'frontmatter.md' heading_level=2 %}
+
+
+
+
+
+
+<!-- This iterates over the controls in a resolved catalog and prints out the control statement, summary table,
+and implemented requirements in a format similar to a FedRAMP .docx
+
+Note that the catalog interface has controls properly sorted so is useful to present the controls how they are organised
+in the original catalog 
+-->
+# Control responses and guidance
+{% for group in catalog_interface.get_all_groups_from_catalog() +%}
+## {{ group.title }} {{ group.class }} \({{ group.id|upper }}\)
```

Comment:
keep in mind some catalogs will have controls not in a group, and the group name for them will be ''